packages/core/src/commands/network-generate.ts

```diff
@@ -46,16 +109,59 @@ export class Command extends Commands.Command {
+        { name: "feeStaticDelegateRegistration", description: "Fee for delegate registration transactions.", schema: Joi.number(), required: false, promptType: "", default: 2500000000 },
+        { name: "feeStaticVote", description: "Fee for vote transactions.", schema: Joi.number(), required: false, promptType: "", default: 100000000 },
+        { name: "feeStaticMultiSignature", description: "Fee for multi signature transactions.", schema: Joi.number(), required: false, promptType: "", default: 500000000 },
+        { name: "feeStaticIpfs", description: "Fee for ipfs transactions.", schema: Joi.number(), required: false, promptType: "", default: 500000000 },
+        { name: "feeStaticMultiPayment", description: "Fee for multi payment transactions.", schema: Joi.number(), required: false, promptType: "", default: 10000000 },
+        { name: "feeStaticDelegateResignation", description: "Fee for delegate resignation transactions.", schema: Joi.number(), required: false, promptType: "", default: 2500000000 },
+        { name: "feeStaticHtlcLock", description: "Fee for HTLC lock transactions.", schema: Joi.number(), required: false, promptType: "", default: 10000000 },
+        { name: "feeStaticHtlcClaim", description: "Fee for HTLC claim transactions.", schema: Joi.number(), required: false, promptType: "", default: 0 },
+        { name: "feeStaticHtlcRefund", description: "Fee for HTLC refund transactions.", schema: Joi.number(), required: false, promptType: "", default: 0 },
+
+        // Env
+        { name: "coreDBHost", description: "Core database host.", schema: Joi.string(), required: false, promptType: "", default: "localhost" },
+        { name: "coreDBPort", description: "Core database port.", schema: Joi.number(), required: false, promptType: "", default: "5432" },
+        { name: "coreDBUsername", description: "Core database username.", schema: Joi.string(), required: false, promptType: "" },
+        { name: "coreDBPassword", description: "Core database password.", schema: Joi.string(), required: false, promptType: "" },
+        { name: "coreDBDatabase", description: "Core database database.", schema: Joi.string(), required: false, promptType: "" },
+
+        { name: "coreP2PPort", description: "Core P2P port.", schema: Joi.number(), required: false, promptType: "", default: 4000 },
+        { name: "coreAPIPort", description: "Core API port.", schema: Joi.number(), required: false, promptType: "", default: 4003 },
+        { name: "coreWebhooksPort", description: "Core Webhooks port.", schema: Joi.number(), required: false, promptType: "", default: 4004 },
+        { name: "coreMonitorPort", description: "Core Webhooks port.", schema: Joi.number(), required: false, promptType: "", default: 4005 },
+
+        // Peers
+        { name: "peers", description: "Peers IP addresses, separated with comma.", schema: Joi.string().allow(""), required: false, promptType: "", default: "127.0.0.1" },
```

Comment:
`Joi.string().allow("")` does this mean that we only allow empty string ? :thinking: or am I missing something

Reply:
No. By default empty flags are now allowed (--peers=""). This way user can set empty flag to skip default option when he want empty list. It also works when user wants to write custom IP addresses, separated with comma.

Reply:
oh ok got it thanks